### DISCUSSION TITLE
winget install option

### DISCUSSION THREAD

**Discussion by sheldonhull** [dagger/dagger#8504] [Ideas]
For windows users, should have a winget install version of the product.
This should reduce friction and probably would make it easier for a windows user to adopt.

**Comment by wingyplus** [comment]
I see it is already there https://github.com/microsoft/winget-pkgs/blob/master/manifests/d/Dagger/Cli/0.13.0/Dagger.Cli.yaml.

And has a PR to put it as an official one at https://github.com/dagger/dagger/pull/7620.
